### DISCUSSION TITLE
希望Bilibili Evolved可以添加批量下载B站视频、批量下载B站视频合集、合并视频的功能。。。

### DISCUSSION THREAD

**Discussion by Meloetta-Nahida** [the1812/Bilibili-Evolved#5008] [功能建议 / Ideas]
现在Bilibili Evolved只可以一个一个下载B站视频，希望Bilibili Evolved将来可以添加批量下载视频、批量下载B站视频合集、合并视频的功能。。。

**Comment by the1812** [comment]
你猜猜这些功能有了没()

**Reply by Geeyun-JY3** [reply]
要不还是在`下载视频`的组件描述里明确写出支持批量下载多 P 视频和合集吧（
